spec/unit/patterns_inspector_spec.rb

```diff
@@ -56,7 +56,7 @@
   describe "#inspect" do
     it "parses the patterns list into a Hash" do
       expect(system).to receive(:run_command).
-        with("zypper", "-xq", "patterns", "-i", {:stdout=>:capture}).
+        with("zypper", "-xq", "--no-refresh", "patterns", "-i", {:stdout=>:capture}).
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.